src/main/java/seedu/address/ui/schedule/ScheduleSlot.java

```diff
@@ -58,4 +63,15 @@ private String getColor(double duration) {
             return "#B8E8FC"; // BLUE
         }
     }
+
+    /**
+     * Change color when user clicks the slot.
+     */
+    @FXML
+    public void handleOnClick() {
```

Comment:
Nice use of java.util.Random to handle color change! However, would it be better for the user to be able to iterate through a fixed set of colors instead since they might want a certain collect and by making it random, it might take a while for the user to get the desired colour?

Reply:
My design here I just to generate random color every time the user clicks it. But you raised a good point!